nextest-runner/src/config.rs

```diff
@@ -309,9 +357,8 @@ struct CustomProfileImpl {
     success_output: Option<TestOutputDisplay>,
     #[serde(default)]
     fail_fast: Option<bool>,
-    #[serde(with = "humantime_serde")]
-    #[serde(default)]
-    slow_timeout: Option<Duration>,
+    #[serde(default, deserialize_with = "deserialize_slow_timeout")]
+    slow_timeout: SlowTimeout,
```

Comment:
This doesn't look right -- what happens if the custom profile doesn't have a slow timeout `period` set? In that case it should inherit the default profile's `period` setting rather than setting it to the default `Duration` (which is zero).